src/Commands/SendSalesCommand.php

```diff
@@ -149,6 +149,7 @@ private function validateConfig(): void
             'api.password' => 'required|string',
             'notifications.mail.name' => 'nullable|string',
             'notifications.mail.email' => 'nullable|email',
+            'notifications.mail.enable_failure_notifications_only' => 'required|boolean',
```

Comment:
```suggestion
            'notifications.mail.trigger_failure_notifications_only' => 'required|boolean',
```

During the final review, I realize changing this word would be helpful. What do you think @hvsharma63 ?

Reply:
Should I make the change in ioi-city-mall package?

Reply:
Yes please